ui/dialog.js

```diff
@@ -337,41 +338,42 @@ return $.widget( "ui.dialog", {
 			})
 			.appendTo( this._appendTo() );
 
-		this._on( this.uiDialog, {
-			keydown: function( event ) {
-				if ( this.options.closeOnEscape && !event.isDefaultPrevented() && event.keyCode &&
-						event.keyCode === $.ui.keyCode.ESCAPE ) {
-					event.preventDefault();
-					this.close( event );
-					return;
-				}
+		this._addClass( this.uiDialog, "ui-dialog", "ui-widget ui-widget-content ui-front" )
+			._on( this.uiDialog, {
```

Comment:
The extra level of indent isn't worth the few characters saved by omitting `;\n\t\tthis.`. Let's revert this to `this._on( ...`.